rfcs/text/0000-planning-for-the-dcp.md

```diff
@@ -0,0 +1,211 @@
+
+## Author(s)
+
+ [Trevor Heathorn](mailto:theathor@ucsc.edu)
+ 
+ [Brian Raymor](mailto:brianraymor@chanzuckerberg.com)
+
+## Shepherd
+
+[TBD](mailto:username@example.com)
+
+## Motivation
+
+***[Note - remove if approved]***
+*Early versions of this RFC were socialized with the DCP community during a PM call and at the DCP meeting in February 2019* 
+
+One of the **challenges** facing the DCP as it matures was identified in this [Update from HCA Science Governance](https://docs.google.com/presentation/d/13hZ7vmr371mN1YFjFyDN-6c0Ul95NwiJcVsjA3nJFVE/edit#slide=id.p3):
+
+> "A lack of reproducible planning process that prioritizes deliverables and ensures confidence that they will meet the needs of HCA"
+
+### DCP _data as a service_
+
+As a starting point, DCP should think *outside-the-box* and adopt one of the [principles of LeSS](https://less.works/less/principles/whole-product-focus.html) 
+>"Customers don’t buy a part of the product, but the whole product … there is no value in separate parts or half-working parts."
+
+Historically, ownership of the DCP software has been distributed across our canonical architectural diagram with its colorful *think-inside-the-boxes* (components). As a result, DCP planning and development have also been distributed to a high degree, resulting in questions or confusion about DCP priorities relative to component priorities with a subsequent lack of alignment for delivery of product increments, unless externally imposed by requests from HCA Science Governance. 
+
+For example, how does a developer know whether to prioritize research for an operational scoping task assigned by the [**Architecture Team**](https://github.com/HumanCellAtlas/dcp-community/blob/master/charters/Architecture/charter.md), collaborate on a cross-component RFC surfaced at a DCP quarterly meeting, or develop a component-specific feature? 
+
+To best serve our users and address the concerns of HCA Science Governance, our plans must focus on the needs and priorities of DCP *data as a service* rather than the individual *Chartreuse* or *Polka Dot* components. This requires a DCP-wide project management process with clear priorities and product increments delivered on a regular cadence. 
+
+## Scientific "guardrails" 
+
+As key stakeholders, the *Oversight Committee* MUST attend Sprint Reviews on a quarterly basis to offer feedback. 
+
+## Detailed Design or _Scrum meets ZenHub_
+
+### **!** Scrum
+
+A single agile framework (referenced as **!** **Scrum**) MUST be used by all component teams for planning and developing DCP *data as a service* in an iterative, incremental manner.
```

Comment:
I would not expect devops and security issues to be emerging continuously - if you're referring to the General Availability DevSecOps requirements. Sam Pierson is currently _rationalizing_ the ZenHub board for those cases, so we can effectively prioritize and schedule these tasks in future Sprints. With regards to _tight feedback loop bugs_ - do you currently buffer a percentage of each Sprint to manage such cases?

I think you meant _we could be much better operating under Kanban than ~agile~ Scrum_? I'm certainly willing to explore other agile (or even waterfall) approaches if there is community consensus. Scrum was selected based on its broad adoption across DCP.  I'm much less familiar with Kanban or its [variations](https://www.atlassian.com/agile/kanban/kanplan) beyond a high-level conceptual overview. Can you outline how it might work across DCP? I have no _ideology_ related to frameworks or applications - just that the DCP  adopts one approach that highlights dependencies, provides tracking and visibility, and addresses the requirements from _Oversight_ for a high-confidence, reproducible process.